### ISSUE TITLE
 Android build tool com.android.tools.build:grade:2.2.0 not generating intermediate "bundle" folder.

### ISSUE THREAD

**Issue by amit-tutu22** [robolectric/robolectric#2674]
### Description

Actually, Roboelectric is expecting asset folder at following location
 build/intermediates/bundles/arm/debug/assets/,  which is not generated by  com.android.tools.build:grade:2.2.0 after switching to AS 2.2.
### Steps to Reproduce

Update to AS2.2 and build tool version - com.android.tools.build:grade:2.2.0
Run existing roboelectric test cases, all test cases are failing due to not finding asset folder.
### Robolectric & Android Version

 testCompile "org.robolectric:robolectric:3.1.2"
build tool version - com.android.tools.build:grade:2.2.0
Android studio 2.2

**Comment by emartynov** [comment]
Is it duplicate for https://github.com/robolectric/robolectric/issues/2647?

**Comment by thierryd** [comment]
I have the same issue and it is NOT fixed by #2647. 

The problem comes from the fact that if you do in Android Studio:

1. Clean project
2. Sync gradle
3. Starts a Robolectric test,

Any test that uses an asset from the assets folder will fail. This comes from the fact that with "com.android.tools.build:gradle:2.1.3", the assets folder in "build\intermediates\assets" was generated when a gradle sync was done. But with "com.android.tools.build:gradle:2.2.3", this is no longer the case.

To work around this problem in Android Studio: build the app (you don't have to deploy it).

The same problem occurs with the command line. If I do a: "gradlew clean test", the assets folder is not generated and any test that uses the asset folder fails. 

To work around this problem on the command line, you can do "gradlew clean assemble test".

But I have another workaround since "assemble" can take a long time. What you have to do is include the dependency ":app:mergeXXXDebugAssets" to ":app:testXXXDebugUnitTest".

I have done so with the following gradle script:

```
project.tasks.whenTaskAdded { task ->
   //Because of https://github.com/robolectric/robolectric/issues/2674, we need to add the dependency of "mergeXXXAssets to "testXXXUnitTest
    if (task.name.startsWith('test') && task.name.endsWith("UnitTest")) {
        String name = task.name;
        String flavorAndBuildTypePrefix = name.substring("test".length());
        String flavorAndBuildType = flavorAndBuildTypePrefix.substring(0, flavorAndBuildTypePrefix.length() - "UnitTest".length());
        String taskId = "merge" + flavorAndBuildType + "Assets"
        task.dependsOn taskId
    }
}
```

I believe #2647 fixed a different issue: it seems to focus on getting the asset for a flavor. But the asset folder is not generated when gradle sync is done.

**Comment by utzcoz** [comment]
Please try the latest Android Studio stable version, and feel free to reopen this issue if it still exists.
